xml/docu_styleguide.structure.xml

```diff
@@ -1577,20 +1577,20 @@ xml:id="tab-install-source"</screen>
    </listitem>
    <listitem>
     <para>
-     If each item in a list consists of a single sentence only, do not use a
-     period at the end of items.
+     Use a period after every list item that is a sentence. Do not use a period 
+     after the items that are not complete sentences.
     </para>
-    <para>
-     If at least one list item consists of two full sentences, end all items
-     in that list with a period.
+    <para>Do not use commas and semicolons to end punctuation. Do not use a full stop 
+     with the last item. Use either all full sentences in your bullet lists 
+     or all fragments. Avoid a mix.
```

Comment:
propose to move the last two sentences up to line 1581, as this info should go together